tox.ini

```diff
@@ -28,6 +28,7 @@ commands =
 
 
 [testenv:lint]
+skip_install=True
```

Comment:
TIL.
But what is it here for?
I don't understand [the docs](https://tox.wiki/en/latest/config.html#skip_install):

> Skip installation of the package. This can be used when you need the virtualenv management but do not want to install the current package into that environment.

Reply:
Can't explain it much better than what was said here: https://chat.canonical.com/canonical/pl/rbqgajaztpyf5g5sdj49maukde